bot/exts/moderation/modpings.py

```diff
@@ -51,10 +52,14 @@ def __init__(self, bot: Bot):
             event_loop=self.bot.loop,
         )
 
-    async def reschedule_roles(self) -> None:
-        """Reschedule moderators role re-apply times."""
+    async def fetch_guild(self) -> None:
+        """Fetch the guild."""
```

Comment:
Maybe specify that it's PyDis we're getting? Probably not needed but could be nice. 

I'd also add that we assign it to `self.guild`.

"Fetch the Python Discord guild and assign to \`self.guild\`."

Reply:
https://github.com/python-discord/bot/pull/2008/commits/1266c5321253da1817d1550efb8008000901a64d